app/assets/javascripts/modules/Contacts.js

```diff
@@ -0,0 +1,62 @@
+moj.Modules.Contacts = {
+    $button : $("#open-button"),
+    $remote_content : $("#remote-content"),
+    $dialog : $("#dialog-content").dialog({
+        autoOpen: false,
+        height: 400,
+        width: 600,
+        modal: true,
+        title: 'Find an address'
+    }),
+
+    init: function() {
+        var self = this;
```

Comment:
I normally only use this if needed to access parent scope - but I understand why you might leave this in.

Reply:
I have removed it here as it's not needed 👍